Test_Automation/result/IPSEC_VPN_log.html

```diff
@@ -0,0 +1,2128 @@
+<!DOCTYPE html>
```

Comment:
All of these result files should not be included. It might help to add a .gitignore so they won't be accidentally added.

Reply:
Hi James,

Thanks for the review. I have specifically put the test result files in result/ directory as a reference. Highlighted the same in README file. The reason for providing the reference results is that when the user trying to test in their test environment, in case of any failures, it will be easy to compare with the existing results. Also, note that the default result files will be of different file names and they will not be overwritten when the script executed.
Please suggest, if you still want to remove the results file.

Thanks,
DineshK
================

Reply:
Happy to hear what other reviewers think, but my feeling is these still don't belong in the repository. The repo won't be updated in sync with DANOS images, so reference results don't make much sense in the repository.

Reply:
I agree, these do not belong in the git repo. The results are a per image, per run artefact. They could be stored in a CI system along side the details run for an image. But they don't belong in git repo, where they will simply become stale.

Reply:
Ok Point taken. I have removed result directory